src/nvidia-install.sh

```
@@ -0,0 +1,2649 @@
+}
+
+# 检测Secure Boot状态
+check_secure_boot() {
+    log_step "$(gettext "secure_boot.check.starting")"
+
+    local secure_boot_enabled=false
+    local secure_boot_method=""
+    
+    # 方法1: 检查/sys/firmware/efi/efivars
+    if [[ -d /sys/firmware/efi/efivars ]]; then
+        if [[ -f /sys/firmware/efi/efivars/SecureBoot-* ]]; then
```

Shellcheck (0.11.0)

[error] -f doesn't work with globs. Use a for loop.

(SC2144)